### ISSUE TITLE
[rulename] issue title

### ISSUE THREAD

**Issue by mahyarmirrashed** [typescript-eslint/typescript-eslint#3773]
- [x] I have tried restarting my IDE and the issue persists.
- [x] I have updated to the latest version of the packages.
- [x] I have [read the FAQ](https://github.com/typescript-eslint/typescript-eslint/blob/master/docs/getting-started/linting/FAQ.md) and my problem is not listed.

**Repro**

#### `.eslintrc.json`
```json
{
  "env": {
    "browser": true,
    "es2021": true
  },
  "extends": [
    "airbnb-base",
    "airbnb-typescript/base",
    "plugin:@typescript-eslint/recommended",
    "prettier"
  ],
  "parser": "@typescript-eslint/parser",
  "parserOptions": {
    "ecmaVersion": 12,
    "project": "tsconfig.json",
    "sourceType": "module"
  },
  "plugins": ["@typescript-eslint"],
  "rules": {
    "@typescript-eslint/explicit-function-return-type": [
      "error",
      {
        "allowExpressions": true
      }
    ],
    "max-len": [
      "warn",
      {
        "code": 80,
        "tabWidth": 2,
        "ignoreUrls": true,
        "ignoreStrings": true,
        "ignoreTemplateLiterals": true
      }
    ],
    "import/extensions": [
      "error",
      "ignorePackages",
      {
        "js": "never",
        "jsx": "never",
        "ts": "never",
        "tsx": "never"
      }
    ],
    "import/no-extraneous-dependencies": 0,
    "react/jsx-filename-extension": 0
  }
}
```

#### Sample `class` providing errors
```ts
export default class Bot extends Client {
  // readonly members
  public readonly commands: Collection<string, Command> = new Collection();
  public readonly events: Collection<string, Event> = new Collection();
  public readonly logger: Consola = console;
}
```

#### `tsconfig.json`
```json
{
  "compilerOptions": {
    "incremental": true,
    "lib": ["esnext"],
    "module": "commonjs",
    "moduleResolution": "node",
    "target": "esnext",
    "outDir": "dist",
    "types": ["node"],
    "strict": true,
    "sourceMap": true,
    "esModuleInterop": true,
    "experimentalDecorators": true,
    "noImplicitAny": true,
    "emitDecoratorMetadata": true,
    "allowSyntheticDefaultImports": true,
    "skipLibCheck": true,
    "skipDefaultLibCheck": true,
    "resolveJsonModule": true
  },
  "include": ["src"],
  "exclude": ["node_modules", "**/*.spec.ts"]
}
```

**Expected Result**

I propose, that for read-only members inside a class, that the rule surrounding spacing should be ignored. The rule should only be applied for methods inside the class.

**Actual Result**

When I do not add the blank lines, I get the following error message inside my IDE (VSCode): `Expected blank line between class members.eslint@typescript-eslint/lines-between-class-members`.

**Additional Info**

There is no additional information.

**Versions**

| package                            | version |
| ---------------------------------- | ------- |
| `@typescript-eslint/eslint-plugin` | `4.29.2` |
| `@typescript-eslint/parser`        | `4.29.2` |
| `TypeScript`                       | `4.3.5` |
| `ESLint`                           | `7.32.0` |
| `node`                             | `16.7.0` |

**Comment by bradzacher** [comment]
Thanks for the issue.
Stylistic rules are frozen and not accepting new options or feature changes.
If you'd like to add this feature for your project - consider forking the rule locally.
